pkg/tests/internal/pulcheck/pulcheck.go

```diff
@@ -29,6 +29,20 @@ import (
 	"gotest.tools/assert"
 )
 
+func resNeedsUpdate(res *schema.Resource) bool {
+	for _, s := range res.Schema {
+		if s.Computed && !s.Optional {
+			continue
+		}
+		if s.ForceNew {
+			continue
+		}
+
+		return res.UpdateContext == nil
```

Comment:
This will return upon the first iteration of the `for` loop. Is that good?

Reply:
not intended, thanks for spotting.